### DISCUSSION TITLE
How to change color link ?

### DISCUSSION THREAD

**Discussion by raphaelchelly** [timlrx/tailwind-nextjs-starter-blog#420] [Q&A] [answered]
Hi, 

Thank you for your work and the quality of the documentation, it is very clear even for a beginner like me. I have a very basic question, I'm trying to change the color of all the links on the blog but I couldn't figure out how to do it. I tried to search in the `tailwind.config.js` file or in the `css` folder but I couldn't find it... Any tips ?

Sorry for this very basic question but thanks in advance if anyone can help me with this.

**Comment by nelsonlaidev** [comment, marked as answer]
You can the edit [this line](https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/tailwind.config.js#L31) to change color.  This is the configuration of the main color.

Example
```diff
  colors: {
-         primary: colors.teal,
+         primary: colors.red,
          gray: colors.neutral,
        },
```

**Reply by raphaelchelly** [reply]
Thanks a lot ! It work 👍
<img width="693" alt="image" src="https://user-images.githubusercontent.com/79171364/158985218-3bc7136a-682f-4789-873c-ba992b6305a4.png">
